acceptance/go.mod

```diff
@@ -5,8 +5,8 @@ go 1.18
 require (
 	github.com/gruntwork-io/terratest v0.31.2
 	github.com/hashicorp/consul-k8s/control-plane v0.0.0-20211207212234-aea9efea5638
-	github.com/hashicorp/consul/api v1.10.1-0.20220614213650-6453375ab228
-	github.com/hashicorp/consul/sdk v0.9.0
+	github.com/hashicorp/consul/api v1.10.1-0.20220719021910-e53a1e4a8a46
```

Comment:
DANGER! Points to my unmerged changes here: https://github.com/hashicorp/consul/pull/13505